doc/how-to/setup.md

```diff
@@ -5,10 +5,24 @@
 
 ## Project dependencies
 
-- NodeJS
-- pnpm: `sudo corepack enable`
+* Version control system [Git](https://git-scm.com)
+	* On MacOS: `brew install git`
+* [NodeJS LTS](https://nodejs.org) (actual in use is 20.17.0)
+	* On MacOS: `brew install node@20`
+* Package manager [pnpm](https://pnpm.io/)
+	* On MacOS: `brew install pnpm`
+	* Enable corepack afterwards : `sudo corepack enable`
```

Comment:
I believe if you have node, you don't have to install pnpm just enable the corepack